website/docs/patterns/disabled-patterns/partials/accessibility.md

```diff
@@ -0,0 +1,31 @@
+The `disabled` attribute in HTML is only valid for these form controls:
+
+- `<button>`
+- `<fieldset>`
+- `<optgroup>`
+- `<option>`
+- `<select>`
+- `<textarea>`
+- `<input>`
+
+In terms of accessibility, however, a form control with the `disabled` attribute is problematic. If a form control has the `disabled` attribute, it's not available to users with assistive technology. It's as though the element does not exist in the DOM at all.
+
+If you wish to present the user with the form they have just filled out, use the `readonly` attribute instead. This will allow the user to receive the correct information about the form they have just filled out, but they won't be able to interact with it.
```

Comment:
**polish (non-blocking):** Showing the user a form they just filled out is only one reason why someone may need to display a non-editable field, so I tried to make this more general. 
```suggestion
When presenting a non-editable field, consider using the `readonly` attribute instead of the `disabled` attribute. This will provide the correct information to all users, but they won't be able to interact with it. 
```